playlist.c

```diff
@@ -107,9 +108,11 @@ typedef int (playlist_sort_fun_t)(
 
 /* TODO/FIXME - hack for allowing the explore view to switch 
  * over to a playlist item */
-void playlist_set_cached(playlist_t* pl)
+void playlist_set_cached_external(playlist_t* pl)
```

Comment:
I'd probably add a NULL check on this argument, just for safety (wouldn't want `NULL->cached_external = true;`)

Reply:
Added